wordpress-mobile/WordPress-Android
# Issue/4659 notifications service

Fixes #4659 

This service implements processing of Like/Approve/Voice reply quick actions.

To test:
1. log in to WPAndroid and log in on another device / a browser with some other account
2. make a comment to a comment/post of yours
3. you'd receive a notification with actions REPLY/APPROVE if your blog is set for comments to be moderated, or REPLY/LIKE if your blog is set for comments to be automatically published
4. If you click APPROVE/LIKE the magic should happen in the background. You'll see a "updating" infinite progress bar notification so you're aware your action is being taken care of.
5. If it goes well, a 3-second notification with a successful message is shown, and the original notification is dismissed.
6. If anything goes wrong (i.e. no internet connection available to actually fulfill your request), a permanent message is shown, and the original notification is kept so you can try again.

While this is the basis for something bigger (#4659), this particularly fixes #4602, and means an enhancement of quick actions (#4523) and voice reply (#4032).
I'll continue to work on #4657 after this one is merged.

cc @daniloercoli @hoverduck
